src/Cake.Common.Tests/Fixtures/Tools/DotNet/Workload/List/DotNetWorkloadListerFixture.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Cake.Common.Tools.DotNet.Workload.List;
+
+namespace Cake.Common.Tests.Fixtures.Tools.DotNet.Workload.List
+{
+    internal sealed class DotNetWorkloadListerFixture : DotNetFixture<DotNetWorkloadListSettings>
+    {
+        public IEnumerable<string> WorkloadIds { get; set; }
+
+        public void GivenInstalledWorkloadsResult()
+        {
+            ProcessRunner.Process.SetStandardOutput(new string[]
+            {
+                "This command lists only workloads that were installed via `dotnet workload install` in this version of the SDK and not those that were installed via Visual Studio.",
+                "",
+                "Installed Workload Ids",
```

Comment:
The output seems to have changed at least with `6.0.400` I get 
```
Installed Workload Ids      Manifest Version      Installation Source
--------------------------------------------------------------------------------------
maccatalyst                 15.4.303/6.0.300      VS 17.3.32804.467, VS 17.4.32804.182
maui-ios                    6.0.312/6.0.300       VS 17.3.32804.467, VS 17.4.32804.182
ios                         15.4.303/6.0.300      VS 17.3.32804.467, VS 17.4.32804.182
maui-maccatalyst            6.0.312/6.0.300       VS 17.3.32804.467, VS 17.4.32804.182
maui-android                6.0.312/6.0.300       VS 17.3.32804.467, VS 17.4.32804.182
maui-windows                6.0.312/6.0.300       VS 17.3.32804.467, VS 17.4.32804.182
android                     32.0.301/6.0.300      VS 17.3.32804.467, VS 17.4.32804.182

Use `dotnet workload search` to find additional workloads to install.
```

So we should have a dotnet workload class instead of just string. ideally based on header
```
Installed Workload Ids      Manifest Version      Installation Source
```
work be tolerant and parser handle new output too.